.github/workflows/helm-app-deploy.yml

```diff
@@ -83,17 +93,37 @@ jobs:
           chmod +x ./kind
           sudo mv ./kind /usr/local/bin/kind
           kind version
+      
+      - name: Create Kind Cluster config
+        id: kind-config
+        run: |
+          MASTERS=${{ github.event.inputs.masters || 1 }}
+          WORKERS=${{ github.event.inputs.workers || 2 }}
+
+          echo "Creating KIND config with $MASTERS masters and $WORKERS workers"
```

Comment:
Please create a separated `scripts/create_kind_config.sh` shell script for this code. It keeps the workflow script shorter and allows to call the script locally for testing.